sporkfish/searcher/move_ordering/composite_heuristic.py

```diff
@@ -0,0 +1,46 @@
+from typing import List
+
+import chess
+
+from ...board.board import Board
+from .killer_move_heuristic import KillerMoveHeuristic
+from .move_order_heuristic import MoveOrderHeuristic, MoveOrderMode
+from .mvv_lva_heuristic import MvvLvaHeuristic
+
+
+class CompositeHeuristic(MvvLvaHeuristic, KillerMoveHeuristic, MoveOrderHeuristic):
```

Comment:
Interesting design - I think this behaviour is (potentially) weird, as you showed me offline, the order of inheritance somehow matters.

Suggest adding an inline comment here for now - we can take a look again later.

Most important thing is it is working, and we get our new feature in.

Reply:
OK for next